frontend/database/00207_add_parent_verification.sql

```diff
@@ -0,0 +1,6 @@
+-- Users table
+ALTER TABLE `Users`
+ADD COLUMN `parent_verified` boolean 
+ADD COLUMN `creation_timestamp` timestamp DEFAULT NULL
```

Comment:
It could be better set the current timestamp as default value:
```suggestion
ADD COLUMN `creation_timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP COMMENT 'Almacena la hora y fecha en que se creó la cuenta de usuario', 
```